tests/test_Consumer.py

```diff
@@ -324,3 +324,38 @@ def test_consumer_without_groupid():
     with pytest.raises(ValueError) as ex:
         TestConsumer({'bootstrap.servers': "mybroker:9092"})
     assert ex.match('group.id must be set')
+
+
+def test_callback_exception_no_system_error():
```

Comment:
I would suggest adding more test cases related to librdkafka layer, C binding layer and python layer.